packages/@okta/vuepress-site/docs/reference/api/linked-objects/index.md

```diff
@@ -61,19 +66,24 @@ If you created multiple User Types (see [User Types](/docs/reference/api/user-ty
 ## Getting started
 
 Explore the Linked Objects API: [![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/3ab2cf3f197337119d34)
+-->
 
 ## Link definition operations
 
-<ApiAuthMethodWarning />
+<!--<ApiAuthMethodWarning />-->
 
-Link definition operations allow you to manage the creation and removal of the link definitions. If you remove a link definition, links based on that definition are unavailable.
+Link definition operations allow you to manage the creation and removal of the link definitions. If you remove a link definition, links based on that definition are unavailable. These operations are available at the new [Okta API reference portal](https://developer.okta.com/docs/api/) as part of the [Linked Objects API](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/LinkedObject/).
```

Comment:
I wonder if we should remove the word "new" - how long until the portal isn't new anymore?

Reply:
Hi @barbaravo-okta . For context, we plan to remove these pages all together once the "new" site becomes the permanent site. We're working on a sunsetting plan, but the "new" won't matter in like 6 months since we'll remove these deprecation pages.

Reply:
Cool, sounds good!